tests/test_self.py

```diff
@@ -847,32 +847,19 @@ def test_import_plugin_from_local_directory_if_pythonpath_cwd(
     ) -> None:
         p_plugin = tmpdir / "plugin.py"
         p_plugin.write("# Some plugin content")
-        with tmpdir.as_cwd():
-            orig_pythonpath = os.environ.get("PYTHONPATH")
-            if sys.platform == "win32":
-                os.environ["PYTHONPATH"] = "."
-            else:
-                os.environ["PYTHONPATH"] = f"{(orig_pythonpath or '').strip(':')}:."
+        if sys.platform == "win32":
+            python_path = "."
```

Comment:
Wondering if we should just skip the test under windows here.

Reply:
Think it makes sense to test both.